packages/tsconfig/tsconfig.base.json

```
@@ -4,6 +4,12 @@
   "extends": "@tsconfig/strictest/tsconfig.json",
   "compilerOptions": {
     "exactOptionalPropertyTypes": false,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "baseUrl": "./",
+    "paths": {
+      "~cms/*": ["../cms/src/*"],
+      "~model/*": ["../model/src/*"]
```

`@nitic-astronomy/jest` と `~cms` があるのパッと見で違いわからないので、やっぱ `@nitic-astronomy/**` で統一したい〜

```suggestion
      "@nitic-astronomy/cms/*": ["../cms/src/*"],
      "@nitic-astronomy/model/*": ["../model/src/*"]
```

---

`package.json`の`exports`フィールドで指定したものと区別できなくなるからやめた方がいいと思う

> `@nitic-astronomy/jest` と `~cms` があるのパッと見で違いわからないので、やっぱ `@nitic-astronomy/**` で統一したい〜

`package.json`の`exports`フィールドで指定したものと区別できなくなるからやめた方がいいと思う

---

んーというと？？

---

あーーなるほどね〜〜〜〜
であれば `~` だと ぱっと見 `~/**` っぽくもあるので、`#` or `#/` がいいっす

---

対応しました😎